nameko/standalone/rpc.py

```diff
@@ -308,6 +317,12 @@ def __getattr__(self, name):
                 self._worker_ctx, name, self._reply_listener)
         return self._proxies[name]
 
+    def __getitem__(self, name):
+        """Enable dict-like access on the proxy. """
+        if not isinstance(name, string_types):
```

Comment:
nice! if you use `getattr(self, name)` instead of calling `self.__getattr__` you get the type check for free